src/pages/index.js

```diff
@@ -13,15 +14,64 @@ import {
   inputSubtitle,
   profileEditBtn,
   addNewImageBtn,
-  initialCards,
   config,
   userInfoObj,
+  serverToken,
+  avatarForm,
+  avatarOverlay,
+  imageDeleteForm,
 } from "../utils/constants.js";
 const user = new UserInfo(userInfoObj);
-//////////////////////////  validators  //////////////////////////
+const handlerFunctions = {
+  handleDelete: function handleDelete(data) {
+    openPopup(popupImageDelete);
+    popupImageDelete.setDeleteSubmitListener(data);
+  },
+  handleLike: function handleLike(data) {
+    apiCall.likeCard(data);
+  },
+  handleImageClick: function handleImageClick(data) {
+    popupWithImage.open(data);
+  },
+};
+const apiFunctionObj = {
+  setUserInfo: function apiSetUserInfoCallback(data) {
+    user.setUserInfo(data);
+  },
+  cardRequest: function apiCardRequestCallback(data) {
+    cardRenderer(gallerySection, data);
+  },
+  setAvatar: function apiSetAvatarCallback(data) {
+    user.setUserAvatar(data);
+  },
+  renderSavingProfileForm: function renderSaving() {
+    popupEditProfileForm.renderSaving(false);
+  },
+  renderSavingAvatarForm: function renderSaving() {
+    popupAvatarForm.renderSaving(false);
+  },
+  renderSavingNewCardForm: function renderSaving() {
+    popupNewCardForm.renderSaving(false);
+  },
+};
+////////////////////////////////////////////////////////////
+///////////////////////    sections     ////////////////////
+////////////////////////////////////////////////////////////
+const gallerySection = new Section(cardRenderer, cardGalleryEL);
+////////////////////////////////////////////////////////////
+///////////////////////   VALIDATORS    ////////////////////
+////////////////////////////////////////////////////////////
 const profileValidator = new FormValidator(config, profileForm);
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object: `formValidators`.  Then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
// define an object for storing validators
const formValidators = {}

const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // Here you get the name of the form (if you don’t have it then you need to add it into each form in `index.html` first)
    const formName = formElement.getAttribute('name')

   // Here you store the validator using the `name` of the form
    formValidators[formName] = validator;
    validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```